src/formElementPlugins/search/src/SummaryView.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { SummaryViewProps } from '../../types'
+import { DisplaySelection, DisplayProps } from './ApplicationView'
+
+const SummaryView: React.FC<SummaryViewProps> = ({ parameters, Markdown, response }) => {
+  const { displayFormat } = parameters
+
+  const displayProps: DisplayProps = {
+    selection: response?.selection,
+    displayFormat,
+    Markdown,
+    isEditable: false,
+  }
+
+  return <DisplaySelection {...displayProps} />
+}
+
+export default SummaryView
```

Comment:
Something strange happening in the Summary view. 

After entering all required fields in a "Features showcase" application, and having added a organisation.registration in the form - Page 3. I got this error:
![Screen Shot 2021-06-01 at 10 29 06 AM](https://user-images.githubusercontent.com/16461988/120246593-c76b2b00-c2c4-11eb-8984-a1223556494a.png)

Showing it's in the Search plugin:
![Screen Shot 2021-06-01 at 10 29 17 AM](https://user-images.githubusercontent.com/16461988/120246548-a4407b80-c2c4-11eb-8721-deb3042a03b2.png)

And the summary view is showing this error for Search plugin (organisation):
![Screen Shot 2021-06-01 at 10 29 45 AM](https://user-images.githubusercontent.com/16461988/120246551-a6a2d580-c2c4-11eb-8020-4f65a7ae0bdb.png)

Reply:
Yup, I know what this it -- the Summary view doesn't have any fallback for "default" display when `displayFormat`is not defined. I'll need to add this.

Reply:
It's also the same problem for the List builder. Will get onto it, thanks for spotting.

Reply:
> It's also the same problem for the List builder. Will get onto it, thanks for spotting.

Actually, it wasn't quite the same problem, for list builder, but similar. Basically, SummaryView failed if `listItems`, `selection` or `displayFormat` wasn't defined. ApplicationView had defaults specified for these, so I've just added the same defaults to SummaryView.

323bd501